README.md

```diff
@@ -11,111 +11,58 @@ This project filters tests based on static analysis of changed files. Changed fi
--  **--local** - Force the project to use local branch and to not pull changes from remote repository. For testing purposes.
--  **--verbose** - Increase verbose level of logging to DEBUG level.
--  **--output-tests** - Outputs only list of tests. Completely turns off all other outputs.
--  **--output-format {raw,markdown}** - Output format selection. Default is **raw**.
-
-### Examples
-**Pull request mode**
-As an example of is used [PR#4957](https://github.com/ComplianceAsCode/content/pull/4957)
 ```
-$ python3 content_test_filtering.py pr 4957
-Changes identified:
-  Profile ospp on rhel7:
-    Rule directory_access_var_log_audit removed from ospp profile.
-  Profile ncp on rhel7:
-    NCP profile extends changed OSPP profile.
-  Profile cui on rhel7:
-    CUI profile extends changed OSPP profile.
-
-Recommended tests to execute:
-    build_product rhel7
-    tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml cui
-    tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ospp
-    tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ncp
+  -h, --help            show this help message and exit
+  --base BASE_BRANCH    Base branch against which we want to compare, default is "master" branch
+  --repository REPOSITORY_PATH
+                        Path to ComplianceAsCode repository. If not provided, the repository will be cloned into
+                        /tmp folder.
+  --remote_repo REMOTE_REPO
+                        Remote repository for pulling, updating and finding branches (Pull requests). Default is
+                        https://github.com/ComplianceAsCode/content.
+  --local               Do not pull from remote, use local branches. Mainly for testing purposes.
+  --verbose             Increase verbose level of logging to DEBUG level. Default level is INFO
+  --output-tests        Output only list of tests. Completely turns off all other outputs.
+  --output-format {raw,markdown}
+                        Output format.
+  --output {commands,json}
+                        Output from the tool.
+  --profile             Print only profile tests.
+  --rule                Print only rule tests.
 ```
 
-**Branch mode**
-Branch mode is similar as Pull request mode. The difference is in arguments used.
-[ComplainceAsCode](https://github.com/ComplianceAsCode/content) project usually doesn't have any branches, so for testing purposes was created [ctf_test_branch](https://github.com/ComplianceAsCode/content/compare/master...mildas:ctf_test_branch) at [forked repository](https://github.com/mildas/scap-security-guide).
+### Remote vs local analysis
+By default, changes are obtained from remote repository. When no options are used, [ComplianceAsCode/content repository](https://github.com/ComplianceAsCode/content/) is clonned, branch with changes fetched, and then localy analysed.
```

Comment:
This is a bit confusing. When no options are used, how does ctf know which branch to fetch and analyze?

Reply:
The `branch` name is needed. The sentence about options was not correctly worded.
I've changed `When no options are used` to `When no optional arguments are passed`

Reply:
Thanks, now it is clear.